README.md

```diff
@@ -127,43 +129,61 @@ pub16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7qssu5l3
   Let's say I have the following recovery phrase
 ```
 $ cat recovery-phrase.txt
-nothing heart matrix fly sleep slogan tomato pulse what roof rail since plastic false enlist
+east student silly already breeze enact seat trade few way online skin grass humble electric
 ```
  Now we create root key
 ```
-$ cat recovery-phrase.txt | cardano-address key from-recovery-phrase Shelley > root.xprv
-xprv1apjwjs3ksgm5mnnk0cc5v5emgv0hmafmmy8tffay5s2ffk69830whwznr46672ruucdzwwtv9upv72e4ylrypyz5m6cyh0p00t7n3u3agt20lv32j4kxcqlkzu78nzjx0ysxxlc2ghfz9prxfmrds802xsh67k7t
+$ cat recovery-phrase.txt \
+| cardano-address key from-recovery-phrase Shelley \
+| tee root.xpr
+
+root_xsk1zz8x2k6jemuq884k4s4c862n2maaskk07ua7xc4pcegfd70fx9ymnhrk5jkex5rh5fggph0682rdxjpaf97lkrmqqs49md3mq0n5ds4e07en8xnyn3g3f85zn85gapcwkht6y5djqrjmqdnqp2rg5nvmyc99drac
 ```
  And derive two signing keys according to multisig CIP (TO-DO add link when available)
 ```
-$ cat root.xprv | cardano-address key child 1852H/1815H/0H/3/0 > signingKey1.xprv
-xprv1nrywvqyuwefulu6mqmpxelz92kcy0zluu5kf5p74c6l6f6z9830pfvcvgen923akys2d2fmlh8t3fttj6nvx040k30ek8k62jyvv9p38c8lyuz9qkyhgnyme7ay742e7nemd037jdevpl688fdvtcfq9eqpnkr57
-$ cat root.xprv | cardano-address key child 1852H/1815H/0H/3/1 > signingKey2.xprv
-xprv1vpr59y3p3cfggk85x6dvmlpkwm9f4c99lvkmw8r6j5vwd669830rw5lvsuh8530q897ht9a297kw2qhkpvn7kk98njhdg3scter64uukrwtashzde54v8ery8a92npx4e22ffg45mtshe6ewpnzjx2cn9qs42k0f
+$ cat root.xprv \
+| cardano-address key child 1852H/1815H/0H/3/0 \
+| tee signingKey1.xprv
+
+script_xsk1tr75z5eem5gxxredy7qj63v95vudfwfrmd0cd0772exekr02x9y6nhnranafyws2d0e5r0du87f0llu4dpdwsrlgxafwlwv7vzsanwa79jjj269v36uqekx368myrrqxe4u86dw0t0ncnjjn6wc8w57pmu5gdsr4
+
+$ cat root.xprv \
+| cardano-address key child 1852H/1815H/0H/3/1 \
+| tee signingKey2.xprv
+
+script_xsk1krdfq60htlc58s6gq4uesytc74vzsa99d9yzpwlyhd2gwzl2x9ya5vhfkt4yzqu4k3v7h3kr00ey983n4er2sl3eftds28hegx5f5hq6x73l4v26e08jcyq3dhwdru6fq9nc54evkktec0hgh5dw7g0uqvnhm24p
 ```
  The corresponding verification keys and their hashes can be obtained as follows
 ```
-$ cat signingKey1.xprv | cardano-address key public --with-chain-code | cardano-address key hash --base16 > verKey1.hash
-de5861cd05e99985b2c586ab383790c6600990809206f84e96eadaea
-$ cat signingKey2.xprv | cardano-address key public --with-chain-code | cardano-address key hash --base16 > verKey2.hash
-aca52d7d28ce353f4766e4e2c8cc2208c7113d794e776eafb8c07a80
-```
-Also notice the default hrp (in bech32) for the hashes of verification key - **xpub_hash**
-```
-$ cat signingKey1.xprv | cardano-address key public --with-chain-code | cardano-address key hash
-xpub_hash1mevxrng9axvctvk9s64nsduscesqnyyqjgr0sn5katdw5egajw2
+$ cat signingKey1.xprv \
+| cardano-address key public --without-chain-code \
```

Comment:
It seems to be impossible to obtain verification key hash of the extended public key:

```
$ cat signingKey1.xprv \
| cardano-address key public --with-chain-code \
| cardano-address key hash \
| tee verKey1.hash

user error (Invalid human-readable prefix. Prefix ought to be one of: ["addr_vk","stake_vk","script_vk"])
```

Is that intentional to enforce compatibility with https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postAnyAddress?

Reply:
It is intentional, to make it clear that the hash is about the public key itself, not the public key + chain code. We could accept both, and simply discard the chain code in the case of the extended one.